docs/plugins/restified-endpoints/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: RESTified Endpoints Plugin
+sidebar_position: 1
+description: "Learn more about the RESTified Endpoints Plugin for Hasura DDN."
+keywords:
+  - hasura plugins
+  - plugins
+  - engine plugins
+  - custom execution
+  - custom endpoints
+  - custom handlers
+  - restified endpoints
+  - restified graphql
+---
+
+# RESTified Endpoints Plugin
+
+The [RESTified Endpoints Plugin](https://github.com/hasura/engine-plugin-restified-endpoint) allows you to add RESTified
+GraphQL endpoints to DDN. This can be used to add custom REST endpoints to DDN that would execute specified GraphQL
+query on the DDN GraphQL API and return the response.
+
+The plugin integrates with Hasura DDN as a **pre-route plugin** and can be deployed as an HTTP service using tools like
```

Comment:
Checking that this can't be hosted on DDN itself?

Reply:
No, this needs to be deployed by the user. We are not going to manage the deployments.